src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/ReferralController.kt

```diff
@@ -274,7 +274,7 @@ class ReferralController(
     var defaultConfirmationFields = ConfirmationFields(
       primaryHeading = "Move referral to ${chosenStatus.description.lowercase()}",
       primaryDescription = "Submitting this will change the status to ${chosenStatus.description.lowercase()}.",
-      secondaryHeading = "Confirm status change",
+      secondaryHeading = chosenStatus.description,
```

Comment:
Shouldn't this be "Assessed as suitable"?

Reply:
The secondary heading now needs to match the status name in the main heading. So for "Move referral to assessed as suitable" for instance, the value needs to be "Assessed as suitable" as you rightly point out. But for "Move referral to on programme" the value needs to be "On programme".